pq/sphincs_sign_verify.c

```diff
@@ -161,8 +161,8 @@ int main(int argc, char** argv)
     }
 
     if (ret == 0) {
-        InitDecodedCert(&decodedCert, cert_der_buf, cert_der_len, 0);
-        ret = ParseCert(&decodedCert, CERT_TYPE, NO_VERIFY, NULL);
+        wc_InitDecodedCert(&decodedCert, cert_der_buf, cert_der_len, 0);
```

Comment:
Note: These API's were added in https://github.com/wolfSSL/wolfssl/pull/4645 and are in v5.1.0 or later.

Reply:
Previously we had to use a macro to expose those private functions. The build macro can be found in asn.h.  I can’t recall the name offhand.

Reply:
See: https://github.com/wolfSSL/wolfssl/blob/e7a121325bbb51560cf954dec66cba069aa199af/wolfssl/wolfcrypt/asn.h#L1944